distribution/manifest.json

```diff
@@ -20,7 +20,8 @@
 		"https://api.github.com/*"
 	],
 	"browser_action": {
-		"default_icon": "icon.png"
+		"default_icon": "icon.png",
+		"default_popup": "options.html"
```

Comment:
This is needed so you can click the extension in Safari to open options.

Reply:
But that changes the behavior 😳 

We don't really want the options to appear in yet another view without `options_ui` styles. Is there any other way? Is it time to generate a Safari-specific manifest?

Also the button currently is linked to GitHub itself, to make it slightly less useless (as people don't generally need to change the options to often)

Reply:
The only time I click the button is to go to options, and it's a bookmark to GitHub which I don't need. 😭

Reply:
I couldn't find any other way to show the RG settings in iOS Safari.

Since Safari is the only extension with manual submission, I think it makes more sense to leave it in and remove the property in a script during the automatic Chrome/Firefox submission.